src/Pages/AddNewTask/AddNewTask.tsx

```diff
@@ -0,0 +1,152 @@
+  const navigate = useNavigate()
+
+  const toast = useContext(ToastContext)
+  const data = useContext(AuthContext)
+  let headers = data?.header
+  //!  Add New Task
+  const handleAdd = (data: INewTask) => {
+    console.log(data);
+    
+    setLoading(true)
+    return baseUrl.post(`/api/v1/Task`, data ,{
+      headers,
+    })
+      .then(() => {
+        if (toast)
+          toast.getToastValue("success", "Task created successfully")
+        setLoading(false)
+        navigate('/dashboard/tasks')
+      })
+      .catch((err) => {
+        if (toast)
+          toast.getToastValue("error", err.response.data.message)
+        setLoading(false)
+      })
+    }
+
+
+
+
+    const { data:employees } = UseAuthenticatedQuery({
+      queryKey: [`getAllTasks`],
+      url: `http://upskilling-egypt.com:3003/api/v1/Users/?groups=2&pageSize=100&pageNumber=1`,
+      config: {
+        headers
+      }
+    })
+
+console.log(employees);
+
+
+
+
+
+    const project = useContext(ProjectContext)
+    let handleGetData = project?.handleGetData
+    let projects = project?.tableData?.data?.data
+  console.log(projects)
+  
+    useEffect(() => {
+      if (handleGetData)
+        handleGetData("get", "",100)
+    }, [])
+
+
+  return <>
+    <main>
+      <div className='bg-white'>
+
+        <h6 className='pt-3 ps-3'>
+          <Link className='text-decoration-none text-black d-flex align-items-center ' to={'/dashboard/tasks'}>
+            <MdKeyboardArrowLeft />
+            <span>
+              View All Tasks
+            </span>
+          </Link>
+        </h6>
+        <h2 className='p-3 pt-0'>Add a New Task</h2>
+
+      </div>
+
+    {employees&&projects? <div className="row justify-content-center mt-4 mb-2">
+        <div className="col-md-8 bg-white simpleModule p-4">
+          <form onSubmit={handleSubmit(handleAdd)}>
+            <label htmlFor="title">Title</label>
+            <input {...register("title", {
+              required
+            })} className="form-control w-100 mt-2 py-2 rounded-4" type="text" placeholder="Title Name" />
+            {errors?.title ? <p className='text-danger mb-0 mt-1'>{errors?.title?.message}</p> : null}
+
+            <label htmlFor="Description" className='mt-3'>Description</label>
+            <textarea {...register("description", {
+              required
+            })} className="form-control w-100 mt-2 mb-1 py-2 rounded-4" placeholder="Description Content" />
+            {errors?.description ? <span className='text-danger mb-0 mt-1'>{errors?.description?.message}</span> : null}
+
+<div className="d-flex justify-content-between align-items-center ">
+
+  <div>
+  <label htmlFor="employeeId" className='mt-3'>User</label>
+
+<select {...register("employeeId", {
+required,
+valueAsNumber: true
+})} className="form-select w-100 mt-2 mb-1 rounded-3"  >
+<option className="text-muted">No Users Selected</option>
+{employees?.data?.map((employee: any) =>
```

Comment:
You can destructure the employee object and get only the values you need as follows:
`{employees?.data?.map(({id,useName}: any) =>{...}`